other/bedwars/thunderbird/map.xml

```diff
@@ -432,6 +447,37 @@
     <kit id="snowball" deduct-items="false">
         <item projectile="throwable-snowball" material="snowball" name="`rSlowball"/>
     </kit>
+    <!-- Store Item Kits (no lore or name) -->
+    <kit id="wool-store" force="true">
+        <item material="wool" team-color="true" amount="16"/>
+    </kit>
+    <kit id="shears-store" force="true">
+        <item material="shears" amount="1"/>
+    </kit>
+    <kit id="glass-store" force="true">
+        <item material="glass" team-color="true" amount="4"/>
+    </kit>
+    <kit id="obsidian-store" force="true">
+        <item material="obsidian" amount="4"/>
+    </kit>
+    <kit id="tnt-store" force="true">
+        <item material="tnt" amount="1"/>
+    </kit>
+    <kit id="wood-store" force="true">
+        <item material="wood" amount="16"/>
+    </kit>
+    <kit id="gapple-store" force="true">
+        <item material="golden apple" amount="1"/>
+    </kit>
+    <kit id="pearl-store" force="true">
+        <item material="ender pearl" amount="1"/>
+    </kit>
+    <kit id="zombie-store" force="true">
+        <item material="monster egg" damage="54" amount="1"/>
+    </kit>
+    <kit id="sponge-store" force="true">
+        <item material="sponge" amount="3"/>
+    </kit>
```

Comment:
I understand that you want to be able to show custom name and lore without making the items unstackable, but I feel that this isn't necessary and would end up making the XML more complex than it needs to be. 

I'd guess that a good majority of players would know the purposes of some of these items (wool, wood planks, sponge, golden apple) even if they've never played the gamemode before, considering that the items have no special functions. It is also unlikely that some of the items would have issues with stacking in the first place (you can't normally obtain ender pearls outside of the shop)